src/test/resources/software/amazon/smithy/lsp/ext/models/operation-name-conflict/b.smithy

```diff
@@ -0,0 +1,16 @@
+$version: "2"
+
+namespace b
+
+string Ignored
```

Comment:
**thought:** this technically can be anything that makes this file longer than the other - which is what ultimately caused the IIOBE (line lookup hitting a different file than intended, which would go over the bounds).